netbeans.apache.org/src/content/about/history.asciidoc

```diff
@@ -0,0 +1,166 @@
+////
+     Licensed to the Apache Software Foundation (ASF) under one
+     or more contributor license agreements.  See the NOTICE file
+     distributed with this work for additional information
+     regarding copyright ownership.  The ASF licenses this file
+     to you under the Apache License, Version 2.0 (the
+     "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing,
+     software distributed under the License is distributed on an
+     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+     KIND, either express or implied.  See the License for the
+     specific language governing permissions and limitations
+     under the License.
+////
+= Apache NetBeans History
+:jbake-type: page
+:jbake-tags: about
+:jbake-status: published
+:keywords: Apache NetBeans History
+:description: Apache NetBeans History
+:toc: left
+:toc-title:
+:icons: font
+
+== Apache NetBeans History
+
+NetBeans started as a student project (originally called Xelfi) in
+Czechoslovakia, now the Czech Republic, in 1996. The goal was to write a
+Delphi-like Java IDE (Integrated Development Environment) in Java. Xelfi was
+the first Java IDE written in Java, with its first pre-releases in 1997. Xelfi
+was a fun project to work on, especially since the Java IDE space was uncharted
+territory at that time.
+
+image::nb-history-1.png[caption="", title="Here you see the original group of students who constituted the original NetBeans Team"]
+
+The project attracted enough interest that the students, once they graduated,
+decided that they could market it as a commercial product. Soliciting resources
+from friends and relatives for a website, they formed a company around it.
+
+Soon after, they were contacted by Roman Stanek, an entrepreneur who had
+already been involved in several startups. He was looking for a good idea to
+invest in, and discovered Xelfi. He met with the founders; they hit it off, and
+a business was born. The original business plan was to develop network-enabled
+JavaBeans components. Jaroslav Tulach, who designed the IDE's basic
+architecture, came up with the name NetBeans to describe what the components
+would do. The IDE would be the way to deliver them. When the specification for
+Enterprise Java Beans came out, it made more sense to work with the standard
+for such components than to compete with it--but the name stuck.
+
+In the spring of 1999, NetBeans DeveloperX2 was released, supporting Swing. The
```

Comment:
X2 supported Swing in `javax.swing` packages. Prior to that we were supporting swing in `com.sun.swing.*` ones with NetBeans Developer 2.0. NetBeans was the first tool on the market to handle the rename of Swing components and that increase the awareness of NetBeans as lot.